README.md

```diff
@@ -31,7 +31,7 @@ but be aware that version will not be updated.
 
       map Alt+Insert kitten kitty_grab/grab.py
 
-* Restart Kitty.
+* Reload config with Ctrl+Shift+f5 or Restart Kitty.
```

Comment:
This assumes default configuration. I’d prefer to say something that acknowledges that the user might have changed the default, like “Reload config (Ctrl+Shift+F5 by default, see `load_config_file`)” (hyperlink to the Kitty configuration docs).

Reply:
Also, keys should be formatted in monospace, see mentions of other keys throughout the file.

Reply:
Overall, thanks for reminding me Kitty now supports config reloading. I think there is at least one more mention of restart down the file, that should probably be changed too.